apps/web/src/pages/EmploymentEquityPage/IndigenousEquityOption.tsx

```diff
@@ -0,0 +1,104 @@
+import React from "react";
+import { useIntl } from "react-intl";
+import { MinusIcon, PlusIcon } from "@heroicons/react/24/outline";
+
+import { Button } from "@common/components";
+
+import { IndigenousCommunity } from "~/api/generated";
+import { IndigenousDialog } from "./dialogs";
+
+import { IndigenousDialogProps } from "./types";
+
+type EquityGroup = "indigenous";
+
+interface EquityOptionProps {
+  indigenousCommunities: Array<IndigenousCommunity>;
+  option: EquityGroup;
+  // Note: Just defining the func signature
+  // eslint-disable-next-line no-unused-vars
+  onSave: (indigenousCommunities: Array<IndigenousCommunity>) => void;
```

Comment:
I wonder if there is an eslint config that would allow us to do this without needthing to ignore? 🤔 

Def don't worry about it in this PR, just dropping some thoughts in 😜

Reply:
That's actually just copy pasta from the original EquityOption.  I tried removing it in [fixes from Eric's review](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5461/commits/e7a019aa742654d22f4aa3f02ab5dd1297ae656c) and eslint didn't complain.  :shrug: